src/POGOProtos/Map/Pokemon/WildPokemon.proto

```diff
@@ -10,5 +10,5 @@ message WildPokemon {
 	double longitude = 4;
 	string spawn_point_id = 5;
 	.POGOProtos.Data.PokemonData pokemon_data = 7;
-	int32 time_till_hidden_ms = 11;
+	int64 time_till_hidden_ms = 11;
```

Comment:
It'll become a problem, when it comes down into that range between 0 and 900000.
I'm not too familiar with the server responses, does that mean it's already being sent as 32bit?